h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -251,9 +247,9 @@ public XGBoostModel(Key<XGBoostModel> selfKey, XGBoostParameters parms, XGBoostO
     params.put("lambda", p._reg_lambda);
     params.put("alpha", p._reg_alpha);
 
-    if (_output.nclasses()==2) {
+    if (output.nclasses()==2) {
```

Comment:
maybe better to use switch here ? It's not important, at some places I just like switches better :)